inference-engine/src/low_precision_transformations/src/convolution_backprop_data.cpp

```diff
@@ -63,7 +63,9 @@ bool ConvolutionBackpropDataTransformation::transform(TransformationContext &con
                          NetworkHelper::getDequantization(reshapeFromWeights);
         if (dequantization.empty()) {
             const auto fqOnWeights = getFakeQuantizeOnWeights(convolutionBackpropData);
-            std::shared_ptr<ngraph::Node> resultConstant = NetworkHelper::fold_fake_quantize(fqOnWeights);
+            auto constantShape = fqOnWeights->input(1).get_shape();
+            std::shared_ptr<ngraph::Node> resultConstant =
+                    NetworkHelper::fold_fake_quantize(fqOnWeights, false, constantShape[1] != 1ul ? 1ul : 0ul);
```

Comment:
```suggestion
            std::shared_ptr<ngraph::Node> resultConstant = NetworkHelper::fold_fake_quantize(fqOnWeights, false, 1ul);
```

Reply:
@eshoguli This check is needed for issue: 56886